src/redturtle/volto/restapi/services/search/get.py

```diff
@@ -61,25 +69,30 @@ def search(self, query=None):
             query = self._parse_query(query)
             queries = []
             term = query.get("SearchableText")
+            indexes_mapping = self.get_indexes_mapping()
+
             for key, value in query.items():
-                # TODO: this must be dynamic
-                if key in ("SearchableText", "Title", "Description"):
+                index_type = indexes_mapping.get(key, None)
+                if index_type == "ZCTextIndex":
+                    # SearchableText, Title, Description
                     queries.append(Eq(key, value))
-                elif key in ("Subject", "portal_type", "tassonomia_argomenti"):
+                elif index_type in ["KeywordIndex", "FieldIndex"]:
                     if isinstance(value, str):
                         value = [value]
                     queries.append(In(key, value))
-                elif key == "path" and isinstance(value, dict):
+                elif index_type == "ExtendedPathIndex":
                     if isinstance(value["query"], list):
                         queries.append(Or(*[Eq(key, p) for p in value["query"]]))
                     else:  # list/tuple ?
                         queries.append(Eq(key, value["query"]))
                 elif key in ("b_start", "b_size"):
                     continue
                 else:
-                    # TODO: skipNull, show_inactive False
-                    logger.warning("Unsupported query parameter: %s %s", key, value)
-                    # return super(SearchHandler, self).search(query)
+                    logger.warning(
+                        f"Unsupported query parameter: {key} {value}. Using standard query."
+                    )
+                    return super(SearchHandler, self).search(query)
```

Comment:
@cekk add a test for this, searching for a index not managed, ie a `dateindex`

Reply:
done